pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -292,6 +292,40 @@ class Build {
 
         return jdkRepo
     }
+
+    private String allocateEBCnodesForTest(testType) {
+        def test_target = buildConfig.TARGET_OS + '_' + buildConfig.ARCHITECTURE
```

Comment:
I think `test_target` is confusing as it is really the platform here. The test target is the testType.